### PR TITLE
Chore: added the file with error texts to the sln for easier debugging

### PR DESCRIPTION
**Pull request by psfinaki**

### CODE REVIEW

File: VisualFSharp.sln [hunk 1]
```diff
@@ -13,6 +13,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "VisualFSharp", "VisualFShar
 	EndProjectSection
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Compiler", "Compiler", "{3881429D-A97A-49EB-B7AE-A82BA5FE9C77}"
+	ProjectSection(SolutionItems) = preProject
+		src\Compiler\FSComp.txt = src\Compiler\FSComp.txt
+	EndProjectSection
```
Comment (vzarytovskii) [inline VisualFSharp.sln:18]: Should also probably add to the service project as a "None" item.

Comment (vzarytovskii) [reply VisualFSharp.sln:18]: And to FSharp.sln and service solutions too.

Comment (vzarytovskii) [reply VisualFSharp.sln:18]: Oh, good.

Comment (psfinaki) [reply VisualFSharp.sln:18]: It's already [somehow](https://github.com/dotnet/fsharp/blob/0d88a9cebc99b5685e2418d6b0981b39bd9d0750/src/Compiler/FSharp.Compiler.Service.fsproj#L96) added here, should we still change this?

Comment (vzarytovskii) [reply VisualFSharp.sln:18]: No, that's ok. Can you add it to other two solutions please?

Comment (psfinaki) [reply VisualFSharp.sln:18]: Sorry the thread wasn't getting updated for me so I wrote the same thing :)
Yep, added to both.

**Review by vzarytovskii — APPROVED** [review]

**Pull request merged**